hal/halx86/amd64/x86bios.c

```diff
@@ -44,71 +46,104 @@ DbgDumpPage(PUCHAR MemBuffer, USHORT Segment)
 VOID
 NTAPI
 HalInitializeBios(
-    _In_ ULONG Unknown,
+    _In_ ULONG Phase,
```

Comment:
I think it's worth to also update [halp.h](https://git.reactos.org/?p=reactos.git;a=blob;f=hal/halx86/include/halp.h;hb=ba447018c873a8fc50b81d929e713546a7b306c7#l549) header file.